storybook/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -67,6 +67,7 @@ const meta = {
   component: Tabs,
   args: {
     children: defaultTabs,
+    onChange: (tabId) => console.log(`Tab ${tabId} was selected.`),
```

Comment:
We generally use [action](https://github.com/Amsterdam/design-system/blob/51640bed21deee54d5dddcbaa22e2213155cd08a/storybook/src/components/Switch/Switch.stories.tsx#L26) for this, no?